pulsar/src/test/java/com/flipkart/varadhi/pulsar/services/PulsarTopicServiceTest.java

```diff
@@ -69,4 +70,14 @@ public void testCreate_ConflictException() throws PulsarAdminException {
         verify(pulsarAdmin.topics(), times(1)).createPartitionedTopic(anyString(), eq(1));
 
     }
+
+    @Test
+    public void testCreate_NewTenantNamespace() throws PulsarAdminException {
+        String newTenant = "testTenantNew";
+        String newNamespace = "testNamespaceNew";
+        PulsarStorageTopic topic = PulsarStorageTopic.from(TEST_TOPIC, CapacityPolicy.getDefault());
+        doNothing().when(topics).createPartitionedTopic(anyString(), eq(1));
+        pulsarTopicService.create(topic, newTenant, newNamespace);
+        verify(topics, times(1)).createPartitionedTopic(eq(topic.getName()), eq(1));
```

Comment:
since test is for checking if new tenant/namespace is getting created along with topic, same should be verified as well ?